src/js/actions/organizationActions.js

```diff
@@ -0,0 +1,7 @@
+import OrganizationsApi from '../api/organizations-api';
+
+const apiUrl = '/api/management/v2';
+const tenantadmApiUrl = `${apiUrl}/tenantadm`;
```

Comment:
👍 good point in having those separated... I'll make sure to bring in the rest of the tenantadm related interactions!

Reply:
good :)